tests/HttpExtractorTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\HttpExtractor\Tests;
+
+use GuzzleHttp\Client;
+use GuzzleHttp\Exception\ClientException;
+use GuzzleHttp\Exception\GuzzleException;
+use GuzzleHttp\Handler\MockHandler;
+use GuzzleHttp\HandlerStack;
+use GuzzleHttp\Middleware;
+use GuzzleHttp\Psr7\Request;
+use GuzzleHttp\Psr7\Response;
+use GuzzleHttp\Psr7\Uri;
+use Keboola\HttpExtractor\HttpExtractor;
+use PHPUnit\Framework\TestCase;
+use function file_get_contents;
```

Comment:
cool, I've not seen this before.  
Just curious what benefit it brings and do you do that with all native methods?